docs/pages/inboxes/disappearing-messages.md

```diff
@@ -2,22 +2,21 @@
 
 Disappearing messages are messages that are intended to be visible to users for only a short period of time. After the message expiration time passes, the messages are removed from the UI and deleted from local storage so the messages are no longer accessible to conversation participants.
 
-Disappearing message behavior is enforced by apps, meaning that apps are responsible for removing messages from their UIs and local storage based on conditions set at the conversation level. Conversation participants using apps that support disappearing messages will have a UX that honors the message expiration conditions. 
+## App-level disappearing messages vs. network-level message expiration
 
-However, it's important to understand that:
+Disappearing message behavior is enforced by apps, meaning that apps are responsible for removing messages from their UIs and local storage based on conditions set at the conversation level. As a feature, disappearing messages doesn't delete messages from the XMTP network. 
 
-- A conversation participant using an app that doesn't support disappearing messages won't experience disappearing message behavior.
-- Messages aren't deleted from the XMTP network.
+Starting with XMTP mainnet, the network will enforce message expiration to delete messages from the network after a retention period currently targeted at 6 months.
```

Comment:
Question: what do you think of mentioning that this is a general condition and is not limited to expired messages?